src/tailwind.css

```diff
@@ -1,8 +1,186 @@
 /* purgecss start ignore */
+
 @tailwind base;
 @tailwind components;
 /* purgecss end ignore */
 
 /* Override utils here*/
 
 @tailwind utilities;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1d2530;
```

Comment:
use tailwind.config.js to add colors, give your colors a name then use `@apply bg-purple` for example